Source/Core/Core/HotkeyManager.cpp

```diff
@@ -179,6 +179,8 @@ constexpr std::array<const char*, NUM_HOTKEYS> s_hotkey_labels{{
     _trans("Undo Save State"),
     _trans("Save State"),
     _trans("Load State"),
+    _trans("Increase Select State Slot"),
+    _trans("Decrease Select State Slot"),
```

Comment:
Grammatically, I think these should be "Increase Select<ins>ed</ins> State Slot"/"Decrease Select<ins>ed</ins> State Slot" (and the same for `HK_INCREMENT_SELECT_STATE_SLOT`/`IncrementSelectStateSlotHotkey`/`IncrementSelectStateSlot` and the decrement ones).

Reply:
Ok, I've made the changes you requested.